src/vmm/src/device_manager/mmio.rs

```diff
@@ -99,7 +99,7 @@ impl MMIODeviceManager {
         &mut self,
         vm: &VmFd,
         mmio_device: devices::virtio::MmioTransport,
-        cmdline: &mut kernel_cmdline::Cmdline,
+        _cmdline: &mut kernel_cmdline::Cmdline,
```

Comment:
How about we create a new function for adding a device to the command line? This way the function is platform independent. The `add_device_to_cmdline` function can be x86_64 specific and only called on `x86_64`.

Reply:
Done, PTAL.